src/IssueViz/IssueVisualizationControl/IssueVisualizationControl.xaml

```diff
@@ -2,8 +2,258 @@
+                       Style="{StaticResource IssueDescriptionDockPanelStyle}">
+                <Grid>
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="Auto" />
+                        <ColumnDefinition Width="*" />
+                    </Grid.ColumnDefinitions>
+                    <imaging:CrispImage Grid.Column="0" 
+                                        Height="16" Width="16"
+                                        Moniker="{Binding Severity, Converter={StaticResource SeverityToMonikerConverter}, Mode=OneWay}" 
+                                        Margin="0,0,5,0"/>
+                    <TextBlock Grid.Column="1" Style="{StaticResource BaseTextBlockStyle}">
+                        <Run Text="{Binding Description, Mode=OneWay}"/>
+                        <Hyperlink NavigateUri="{Binding RuleHelpLink}" 
+                                   RequestNavigate="Hyperlink_OnRequestNavigate"
+                                   ToolTip="{Binding RuleKey, StringFormat={}(Open description of rule {0}), Mode=OneWay}">
+                            <Run Text="{Binding RuleKey, StringFormat={}({0}), Mode=OneWay}" />
+                        </Hyperlink>
+                    </TextBlock>
+                </Grid>
+            </DockPanel>
+
+            <ListView Name="LocationsList" 
+                      Grid.Row="1"
+                      Style="{StaticResource BaseListViewStyle}"
+                      ItemContainerStyle="{StaticResource LocationsListViewItemStyle}"
+                      ItemsSource="{Binding LocationListItems}"
+                      SelectedItem="{Binding CurrentLocationListItem, Mode=TwoWay}">
+                <ListBox.Resources>
+                    <DataTemplate DataType="{x:Type viewModels:FileNameLocationListItem}">
+                        <Grid>
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Name="FileType" Width="Auto"/>
+                                <ColumnDefinition Name="FileName" Width="*"/>
+                            </Grid.ColumnDefinitions>
+                            <imaging:CrispImage Grid.Column="0" 
+                                            Height="16" Width="16"
+                                            Moniker="{Binding Icon}"/>
+
+                            <TextBlock Grid.Column="1" 
+                                       Style="{StaticResource FileNameLocationListItemTextBlockStyle}" 
+                                       ToolTip="{Binding FullPath}"
+                                       Text="{Binding FileName}" />
+                        </Grid>
+                    </DataTemplate>
+                    <DataTemplate DataType="{x:Type viewModels:LocationListItem}">
+                        <Grid Margin="10,0,0,0">
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Name="IndentIcon" Width="Auto"/>
```

Comment:
Visually this icon looks like an expander in a hierarchy, but it isn't.  What's it purpose? To make the relationship to the file more obvious>

Reply:
Yep, it felt a bit blank without it. I tried to put the line like in SonarQube, but it looked like crap.
If you got time, maybe you can install Mads' [KnownMonikers explorer](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.KnownMonikersExplorer) and see if there's anything that looks better?
Maybe something like `OverlayTimelineMarkGray` or `InitialNode` could work better, I don't know.

Reply:
OverlayTimelineMarkGray 
![image](https://user-images.githubusercontent.com/60586879/90423570-01b06680-e0bd-11ea-8ca5-e96d4577ca3d.png)

InitialNode
![image](https://user-images.githubusercontent.com/60586879/90423636-1bea4480-e0bd-11ea-96f7-8a9b817d1c7f.png)

Reply:
Maybe ask Chris. I think it looks better without anything.
For me, having a image that looks like an expander but isn't is too confusing (and irritating if you try clicking on it when nothing happens).

Reply:
FYI, Chris also said without anything, so I removed it